src/routes/secured/chat.ts

```diff
@@ -0,0 +1,82 @@
+import express, { Response, Request } from 'express';
+
+import ResponseValue from '../../utils/ResponseValue';
+import { asyncHandler } from '../../middleware/async-handler';
+import { MovnetError } from '../../middleware/MovnetError';
+import { ChatService } from '../../services/chat';
+import { RequestWithUser } from '../../interfaces/requests';
+
+module.exports = (router: express.Router) => {
+  /**
+  * Adds a new chat
+  */
+  router.post('/chat', asyncHandler(async (req: Request, res: Response) => {
+    const name = req.body.name;
+    const event_id = req.body.event_id;
+
+    if(!name || !event_id){
+      throw new MovnetError(422, 'All fields must be filled out');
+    }
+
+    const chat = await ChatService.add(name, event_id);
+
+    return res.json(new ResponseValue(true, chat));
+  }));
+
+  /**
+  * Creates a message and adds it to chat
+  */
+  router.post('/chat/:chat_id', asyncHandler(async (req: Request, res: Response) => {
+    const chat_id = req.params.chat_id;
```

Comment:
Why not put `chat_id` in the body as well?

Reply:
The route `POST: /api/chat` is already taken, so I used `POST: /api/chat/:chat_id`

Reply:
Ok I fixed this, and updated the documentation to reflect it